### PR TITLE
Get user by userName

### PR DESCRIPTION
**Pull request by HubertTatar**

### CODE REVIEW

File: functional_test/live_tests/users_test.py [hunk 1]
```diff
@@ -36,3 +36,8 @@ def test_signup_existing_user(pet_store_client):
     response = pet_store_client.signup_user(user)
     assert_that(response.status_code, is_(409))
 
+def test_user_by_username(pet_store_client):
+    response = pet_store_client.find_user_by_name('jwick201')
+
+    pets = response.json()
+    assert_that(pets['userName'], is_('jwick201'))
```
Comment (pauljamescleary) [inline functional_test/live_tests/users_test.py]: add a new line

File: functional_test/pet_store_client.py [hunk 2]
```diff
@@ -133,3 +133,9 @@ def signup_user(self, user):
         return self.session.request('POST', url, self.headers, json.dumps(user))
 
 
+    def find_user_by_name(self, userName):
+        """
+        Get user by userName
+        """
+        url = urljoin(self.index_url, "/users/{0}".format(userName))
+        return self.session.request('GET', url, self.headers)
```
Comment (pauljamescleary) [inline functional_test/pet_store_client.py]: add a newline

File: functional_test/live_tests/users_test.py (hunk 1 above)
Comment (HubertTatar) [reply functional_test/live_tests/users_test.py]: done

File: functional_test/pet_store_client.py (hunk 2 above)
Comment (HubertTatar) [reply functional_test/pet_store_client.py]: done

Comment (HubertTatar) [reply functional_test/pet_store_client.py]: Just realised that you propably ment to add new line after assert and return. Is that right ?

File: functional_test/live_tests/users_test.py [hunk 3]
```diff
@@ -36,3 +36,9 @@ def test_signup_existing_user(pet_store_client):
     response = pet_store_client.signup_user(user)
     assert_that(response.status_code, is_(409))
 
+def test_user_by_username(pet_store_client):
+    response = pet_store_client.find_user_by_name('jwick201')
+
+    user = response.json()
+
+    assert_that(user['userName'], is_('jwick201'))
```
Comment (pauljamescleary) [inline functional_test/live_tests/users_test.py]: needs newline

File: functional_test/pet_store_client.py [hunk 4]
```diff
@@ -133,3 +133,10 @@ def signup_user(self, user):
         return self.session.request('POST', url, self.headers, json.dumps(user))
 
 
+    def find_user_by_name(self, userName):
+        """
+        Get user by userName
+        """
+        url = urljoin(self.index_url, "/users/{0}".format(userName))
+
+        return self.session.request('GET', url, self.headers)
```
Comment (pauljamescleary) [inline functional_test/pet_store_client.py]: needs newline

**Comment by pauljamescleary** [comment]
Can merge once travis is happy

**Comment by HubertTatar** [comment]
Done.

**Comment by pauljamescleary** [comment]
:champagne:  Thanks a ton for the contribution!

**Pull request merged**